conf/sample.config.inc.php

```diff
@@ -33,11 +33,11 @@
  *
  */
 $conf['datasources']['localhost'] = array(
-	'host'	=> 'localhost',
+	'host'	=> getenv('DB_PORT_3306_TCP_ADDR') ?: 'localhost',
```

Comment:
I'll have to double check but I believe the link in `docker-compose` sets the ENV prefix but the rest of it is predefined / not editable

```
  links:
    - mysql:db
```

so I think we could make it `ANEMOMETER_DB_PORT_3306_TCP_ADDR` and then it will still work as a solo docker container and within docker compose by making docker compose
```
  links:
    - mysql:anemometer
```